lib/coinbase/pagination.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module Coinbase
+  # A module of helper methods for paginating through resources.
+  module Pagination
```

Comment:
If were calling this like a class, I think it would make sense to be just a class.

Reply:
Let's pair on how we can make this the most readable + useful as we add more paginated endpoints!